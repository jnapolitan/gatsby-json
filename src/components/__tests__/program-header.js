import React from "react";
import renderer from "react-test-renderer";
import ProgramHeader from "../program-header";

describe("Program Header", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ProgramHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});