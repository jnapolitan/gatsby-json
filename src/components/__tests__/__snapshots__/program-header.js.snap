// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Program Header renders correctly 1`] = `
<div
  className="container"
>
  <h3 />
  <button
    className="modalButton"
    onClick={[Function]}
  >
    Learn more
  </button>
</div>
`;
